src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -23,7 +23,15 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
 
   val libraryServiceConstructor: UserInfo => LibraryService
 
+  val duosAutocompleteUrl = FireCloudConfig.Duos.baseUrl + "/autocomplete?types=disease&q=%s"
+
   val libraryRoutes: Route =
+    path("duos" / "autocomplete" / Segment) { (searchTerm) =>
+      get { requestContext =>
+        val extReq = Get(duosAutocompleteUrl.format(searchTerm))
```

Comment:
```
        val extReq = Get(duosAutocompleteUrl.format(java.net.URLEncoder.encode(searchTerm,"UTF-8")))
```
fixes the problem with spaces ... but there might be a cleaner approach in which you create a `Uri` to use as the argument to `Get`, instead of creating a `String`.